.gitignore

```diff
@@ -2,4 +2,5 @@ public/assets/
 bin/
 package-lock.json
 node_modules/
-.sass-cache/
\ No newline at end of file
+.sass-cache/
+.idea/
```

Comment:
👀

Reply:
Just ignoring the folder created by Goland IDE 😅